### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by MarwanElkhoudary**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
-# sahemProject
\ No newline at end of file
+# sahemProject
+
+
+### About 
+sahem is a platform that links creative youth from Gaza strip who lake the suffecient fund to start their own projects with donors from all around the world.
```
Comment (ashatat) [inline README.md]: I think you meant lack, not lake?

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+
+---
+
+### Problem statement:
+The existence of guys with alot of creative and development ideas who lacke the required money to fund thier projects while on the other hand, there are alot of people all around the world who are interesting in gaza economy development and are willing to support people with creative ideas.
```
Comment (ashatat) [inline README.md]: same thing lack, their 

please when you are not sure use google translator

File: README.md [hunk 3]
```diff
@@ -1 +1,54 @@
...
+
+### Our solution:
+Creating a website that grants creative people the opportionity to create and present thier ideas globally and be given the chance to be funded while at the same time allow donors who are interested in development in gaza from all around the world to view these projects, search with their criteria of interest and fund their desirable projects.
+
+---
+
+### User Story
+* As a user I want to be able to register at the website.
+* As a user I want to be able to login to my account.
+* As a register user I want to be able to create project proposal.
```
Comment (ashatat) [inline README.md]: Is there a difference between a user and a registered user?

Comment (othman-shamla) [reply README.md]: user is the guest and i will edit it

**Comment by othman-shamla** [comment]
@ashatat 
we fix the typo

**Pull request merged**